organization-bootstrap/business-units/main.tf

```diff
@@ -0,0 +1,162 @@
+  tf_project_id                 = module.project-tf.project_id
+  top_level_folder_name         = var.business_unit_1_name
+  second_level_folders_names    = var.business_unit_1_envs
+  generate_service_account_keys = var.generate_service_account_keys
+  gcs_location                  = var.gcs_location
+  organization_id               = var.organization_id
+  prefix                        = var.prefix
+  root_node                     = var.root_node
+
+}
+
+# Business unit 2
+
+module "business-unit-2-folders-tree" {
+  source = "./modules/business-unit-folders-tree"
+
+  billing_account_id            = var.billing_account_id
+  tf_project_id                 = module.project-tf.project_id
+  top_level_folder_name         = var.business_unit_2_name
+  second_level_folders_names    = var.business_unit_2_envs
+  generate_service_account_keys = var.generate_service_account_keys
+  gcs_location                  = var.gcs_location
+  organization_id               = var.organization_id
+  prefix                        = var.prefix
+  root_node                     = var.root_node
+
+}
+
+# Business unit 3
+
+module "business-unit-3-folders-tree" {
+  source = "./modules/business-unit-folders-tree"
+
+  billing_account_id            = var.billing_account_id
+  tf_project_id                 = module.project-tf.project_id
+  top_level_folder_name         = var.business_unit_3_name
+  second_level_folders_names    = var.business_unit_3_envs
+  generate_service_account_keys = var.generate_service_account_keys
+  gcs_location                  = var.gcs_location
+  organization_id               = var.organization_id
+  prefix                        = var.prefix
+  root_node                     = var.root_node
+
+}
+
+###############################################################################
+#                              Audit log exports                              #
+###############################################################################
+
+# Audit logs project
+
+module "project-audit" {
+  source          = "terraform-google-modules/project-factory/google//modules/fabric-project"
+  version         = "3.2.0"
+  parent          = var.root_node
+  billing_account = var.billing_account_id
+  prefix          = var.prefix
+  name            = "audit"
+  lien_reason     = "audit"
+  activate_apis   = var.project_services
+  viewers         = var.audit_viewers
+}
+
+# Audit logs destination on BigQuery
+
+module "bq-audit-export" {
+  source                   = "terraform-google-modules/log-export/google//modules/bigquery"
+  version                  = "3.0.0"
+  project_id               = module.project-audit.project_id
+  dataset_name             = "logs_audit_${replace(var.business_unit_1_name, "-", "_")}"
+  log_sink_writer_identity = module.log-sink-audit.writer_identity
+}
+
+# Audit log sink for business unit 1
+# set the organization as parent to export audit logs for all business units
```

Comment:
Audit log sink should be on the environment, not the bu. No idea really on how to do it unless the log module adds multiple support, so in this example it's probably best to switch to a single org-level export.